Sources/Actions/Admin/Reactions.php

```diff
@@ -285,6 +287,39 @@ public function editreactions(): void
 			'class' => 'centertext',
 		];
 
+		// Add a row for a blank field to add a reaction, and a link to add another blank field.
+		$listOptions['additional_rows'][] = [
+			[
+				'position' => 'bottom_of_list',
+				'data' => [
+					'value' => '<input type="text" name="reacts_add[]">'
+				]
+			],
+			[
+				// Clicking this magic link adds a new row...
+				'position' => 'bottom_of_list',
+				'data' => [
+					'value' => '<a href="javascript:void(0);" onclick="addrow()">' . Lang::$txt['reacts_add'] . '</a>'
```

Comment:
Can you make this a <button type="button" instead so we don't have to come back to it when fixing for wcag sutff

Reply:
Sure. I'm just writing up basic code at this point and haven't even tested any of this yet (I plan to start that soon so I don't end up with a ton of stuff to fix later)